File: java/model/B/ClassC.java
package model.B;

import java.io.Serializable;
import org.json.*;
import java.util.*;

public class ClassC implements Serializable{

    public ClassC(){}    private int var1;
    private String var2;


    public ClassC(JSONObject dict) throws JSONException{
        this.var1 = dict.getInt("var1");
        this.var2 = dict.getString("var2");
    }

    public void setVar1(int var1){
        this.var1=var1;
    }

    public int getVar1(){
        return this.var1;
    }

    public void setVar2(String var2){
        this.var2=var2;
    }

    public String getVar2(){
        return this.var2;
    }

    public JSONObject toDict() throws JSONException{
        JSONObject dict = new JSONObject();
        dict.put("var1", String.valueOf(this.var1));
        dict.put("var2", String.valueOf(this.var2));
        return dict;
    }
}

File: swift/model/B/ClassC.swift

import UIKit


class ClassC{

    var var1 = Int()
    var var2 = String()


    init(){}

    init(dict:Dictionary<String, String>){
        self.var1 = Int(dict["var1"]!)!
        self.var2 = dict["var2"]!
    }

    func setVar1(var1:Int){
        self.var1=var1
    }

    func getVar1()->Int{
        return self.var1
    }

    func setVar2(var2:String){
        self.var2=var2
    }

    func getVar2()->String{
        return self.var2
    }

    func toDict()->Dictionary<String, String>{
        var dict = Dictionary<String, String>()
        dict["var1"] = String(self.var1)
        dict["var2"] = String(self.var2)
        return dict
    }
}
